README.md

```diff
@@ -231,6 +232,16 @@ most recent successful snapshot:
 Note: the `GHE_DATA_DIR` variable set in `backup.config` can be used to change
 the disk location where snapshots are written.
 
+### How do backup snapshots differ from the HA (High Availability) replica?
+The backup utilities and [HA replica](https://help.github.com/enterprise/admin/guides/installation/high-availability-cluster-configuration/) serve different yet complementary purposes as part of a recommended GitHub Enterprise deployment.
+
+##### The purpose of the High Avaibility replica
+The HA replica is a fully redundant secondary GitHub Enterprise instance, kept in sync with the primary instance via replication of all major datastores. This active/passive cluster configuration is designed to minimize service disruption in the event of hardware failure or major network outage affecting the primary instance. Because some forms of data corruption or loss may be replicated immediately from primary to replica, it is not a replacement for the backup utilities as part of your disaster recovery plan.
```

Comment:
`The HA replica replicates all major datastore from the primary instance to minimize service disruption in even of a primary instance outage. As it may replicate data loss/corruption it is not a disaster recovery solution and the backup utilities should be used as well`.